api/origin_trials_api.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from framework import basehandlers
+import requests
+import settings
+
+
+class OriginTrialsAPI(basehandlers.APIHandler):
+
+  def do_get(self, **kwargs):
+    """Get a list of all origin trials."""
+    key = settings.OT_API_KEY
```

Comment:
If we are planning on adding more requests besides the GET, we could move all the OT client requests logic to a new file in `frameworks`. Then we can cleanly separate OT client code from the server handler code.

Reply:
I want to make sure I'm understanding correctly here - We want to separate code for handling client-side requests vs. server side requests like automated trial creation, is that right? If so, would it be better to keep the server handler code in `framework` instead, and keep the client API request handlers in this file? I'm not certain of the best organization here

Reply:
api/origin_trials_api.py should handle the incoming request to the ChromeStatus API and returning an appropriate response.

framework/origin_trials_client.py should handle sending any requests to the OT API.

The workflow would be like this
1. incoming request to the Chromestatus API which is handled by `api/origin_trials_api.py`
2. `api/origin_trials_api.py` parses the request and validates any passed in data (Not for this current GET method but you can imagine in the POST method)
3. `api/origin_trials_api.py` calls `framework/origin_trials_client.py` to do some operation
4. `framework/origin_trials_client.py` calls framework/secrets to get the OT secret and then makes the appropriate call the OT API. (In the future, `framework/origin_trials_client.py`, could raise certain custom exceptions which could be caught by `api/origin_trials_api.py` to let it know which http error to return)
5. `api/origin_trials_api.py` gets the returned value and sends the response back

Reply:
I see, that makes sense - I'll try and implement this structure